docs/design/security.md

```diff
@@ -93,12 +93,12 @@ A pod runs in a *security context* under a *service account* that is defined by
 
 ### Related design discussion
 
-* Authorization and authentication http://docs.k8s.io/design/access.md
-* Secret distribution via files https://github.com/GoogleCloudPlatform/kubernetes/pull/2030
-* Docker secrets https://github.com/docker/docker/pull/6697
-* Docker vault https://github.com/docker/docker/issues/10310
-* Service Accounts: http://docs.k8s.io/design/service_accounts.md
-* Secret volumes https://github.com/GoogleCloudPlatform/kubernetes/4126
+* [Authorization and authentication](./access.md)
+* [Secret distribution via files](https://github.com/GoogleCloudPlatform/kubernetes/pull/2030)
+* [Docker secrets](https://github.com/docker/docker/pull/6697)
+* [Docker vault](https://github.com/docker/docker/issues/10310)
+* [Service Accounts:](./service_accounts.md)
+* [Secret volumes](https://github.com/GoogleCloudPlatform/kubernetes/4126)
```

Comment:
Broken. I think this URL should instead be: https://github.com/GoogleCloudPlatform/kubernetes/pull/4126